cmd/enry/main.go

```diff
@@ -29,7 +29,7 @@ func main() {
 	breakdownFlag := flag.Bool("breakdown", false, "")
 	jsonFlag := flag.Bool("json", false, "")
 	showVersion := flag.Bool("version", false, "Show the enry version information")
-	onlyProg := flag.Bool("prog", false, "Only show programming file types in output")
+	allLangs := flag.Bool("all", false, "Show not only the files with programming languages (default) but all languages instead")
```

Comment:
I find the semantics of the description a little confusing. Is the following what you mean?

> Show all files, including those marked as non-programming languages

?